### DISCUSSION TITLE
Change to custom font firefox android

### DISCUSSION THREAD

**Discussion by Kai-Nui** [darkreader/darkreader#9858] [Q&A]
I tried changing font by selecting one from the list but some options fallback to default. Also there is no check mark when adding custom font by typing in font name. Am I missing something?

**Comment by Gusted** [comment]
Hi @Kai-Nui!

> I tried changing font by selecting one from the list but some options fallback to default.

Which options are falling back? 

> Also there is no check mark when adding custom font by typing in font name.

There indeed is none, once you exit the input field the value will be saved. Unfortunately the experience for this is not that great on mobile.
